src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -907,7 +907,7 @@
     <value>foreach requires that the return type '{0}' of '{1}' must have a suitable public MoveNext method and public Current property</value>
   </data>
   <data name="ERR_BadGetAsyncEnumerator" xml:space="preserve">
-    <value>Async foreach requires that the return type '{0}' of '{1}' must have suitable public WaitForNextAsync and TryGetNext methods</value>
+    <value>Async foreach requires that the return type '{0}' of '{1}' must have a suitable public MoveNextAsync method and public Current property</value>
```

Comment:
>MoveNextAsync method and public Current [](start = 98, length = 39)

Please use single-tick quotes around the identifiers 'MoveNextAsync' and 'Current'

Reply:
I updated this string as well as others that had the same problem.

---
In reply to: [223822517](https://github.com/dotnet/roslyn/pull/30280#discussion_r223822517) [](ancestors = 223822517)